src/main/java/uk/ac/ebi/ena/readtools/sampler/FastqFileReader.java

```diff
@@ -48,21 +48,21 @@
     private boolean 
     read( InputStream             is, 
           final QualityNormalizer normalizer,
-          final String stream_marker ) throws SecurityException, DataFeederException, NoSuchMethodException, IOException
+          final String stream_marker ) throws SecurityException, DataProducerException, NoSuchMethodException, IOException

```

Comment:
Make exceptions unchecked and remove throws clause. Delegate quality normalisation to Picard.